The command OPERA-MS check_dependency is executed as a sanity check. It will
however revert an earlier created symlink to the correct perl interpreter.
This patch makes sure the existing correct symlink is not removed.

steven.vandenbrande@kuleuven.be
--- OPERA-MS.pl.orig	2025-04-07 14:03:09.361405000 +0200
+++ OPERA-MS.pl	2025-04-07 14:03:56.109454000 +0200
@@ -278,11 +278,11 @@
     #check_software("$opera_ms_dir/bin", "bundler", $sofware_name, $test_command, $opera_ms_dependency) = @_;
     
     #For the perl verion
-    if($stage eq "CHECK_DEPENDENCY"){
-	my $perl_path = "$utils_dir/perl";
-	`rm $perl_path` if(-e "$perl_path");
-	`ln -s $^X $perl_path`;
-    }
+    #if($stage eq "CHECK_DEPENDENCY"){
+    #	my $perl_path = "$utils_dir/perl";
+    #	`rm $perl_path` if(-e "$perl_path");
+    #	`ln -s $^X $perl_path`;
+    #}
     $opera_ms_dependency->{"perl"} = $utils_dir;
     
     #OPERA-LG
